Dockerfile

```diff
@@ -1,11 +1,32 @@
-FROM node:16.13.1-alpine3.15
+# syntax=docker/dockerfile:1
+FROM node:lts-alpine3.17 as builder
```

Comment:
Wouldn't it be better to specify the exact version so if someone else pulls the dockerfile later on doesn't get a different minor version?

Reply:
Also, what were the benefits of having a builder stage? Wouldn't it mean that the builder stage is not cached and every release would have to reinstall the node modules part even if it's just a change in a string in our code?